src/main/java/com/sprint/findex_team6/repository/IndexValRepository.java

```diff
@@ -13,15 +13,15 @@
 
 @Repository
 public interface IndexValRepository extends JpaRepository<IndexVal, Long> {
-  Optional<IndexVal> findByIndex_IdAndBaseDate(Long index_id, LocalDate baseDate);
-  
-  Optional<IndexVal> findAllByIndex_IdAndDate(Long indexId, LocalDate date);
+
+  List<IndexVal> findByIndex_Id(Long id);
+
+  Optional<IndexVal> findAllByIndex_IdAndBaseDate(Long indexId, LocalDate date);
```

Comment:
```suggestion
  Optional<IndexVal> findByIndex_IdAndBaseDate(Long indexId, LocalDate date);
```
아 이부분 하나만 가져오는 게 보장되기 때문에이렇게 써도 되지 않을까요?

Reply:
그러네요 감사합니다